### DISCUSSION TITLE
v0.12.0 release date?

### DISCUSSION THREAD

**Discussion by fede843** [nginx/nginx-prometheus-exporter#524] [General]
Hi all! 
Is there an estimation on the release date of version 0.12.0 ? Milestone chart seems completed https://github.com/nginxinc/nginx-prometheus-exporter/milestone/6

**Comment by lucacome** [comment]
Hi @fede843 

The next release will be `v1.0.0`, I've updated the [milestone](https://github.com/nginxinc/nginx-prometheus-exporter/milestone/7).
 We don't have a date yet, there still a few tasks left before the release.

**Reply by fede843** [reply]
OK, that is fine. I was interested on the logging formatting, that is included in 0.12. I guess I'll have to wait then.
Thanks for the update!
